src/components/Button/Button.scss

```diff
@@ -1,12 +1,13 @@
 .button {
-  color: white;
-  border: 1px solid transparent;
-  border-bottom-color: white;
+  color: rgba(255, 255, 255, 0.2);
+  display: block;
+  //border: 1px solid transparent;
+  //border-bottom-color: white;
   padding: 10px 30px;
   transition: border 0.25s;
 
-  &:hover {
-    border-color: white;
-    cursor: pointer;
+  &:hover,
+  &:focus {
```

Comment:
I think we should make a general outline mixin for focus, I don't think hover and focus should have the same styles

Reply:
:+1: